lib/hocon/impl/simple_config_list.rb

```diff
@@ -21,6 +24,8 @@ def initialize(origin, value, status = ResolveStatus.from_values(value))
     end
   end
 
+  def_delegators :@value, :[], :include?, :empty?, :size, :index, :rindex, :each, :map
+
```

Comment:
I don't know what to make of this `Forwardable` stuff.  I'm scared of it.  It doesn't seem to correspond to anything in the java version?